client.go

```diff
@@ -152,20 +152,20 @@ func (c *Client) StartFromInvitation(link string) error {
 
 func (c *Client) GetDirectoryTree() ([]*userFolder, error) {
 	var res []*userFolder
-	if err := c.model.Find(&res, nil); err != nil {
+	if err := c.collection.Find(&res, nil); err != nil {
 		return nil, err
 
 	}
 	return res, nil
 }
 
 func (c *Client) InviteLinks() ([]string, error) {
-	host := c.store.Service().Host()
-	tid, _, err := c.store.ThreadID()
+	host := c.db.Service().Host()
+	tid, _, err := c.db.ThreadID()
 	if err != nil {
 		return nil, err
 	}
-	tinfo, err := c.store.Service().Store().ThreadInfo(tid)
+	tinfo, err := c.db.Service().GetThread(context.Background(), tid)
```

Comment:
`GetThread` takes a `Context`, there's not one around to use, so just passing `context.Background()`. Maybe it should be something else?

Reply:
Shouldn't be a problem, but maybe some context with a timeout would be better... to fail a bit faster if something goes wrong.